db/data.py

```diff
@@ -106,6 +106,15 @@ def submit_low_level_data(mbid, data):
         else:
             data['metadata']['audio_properties']['lossless'] = False
 
+        if 'is_mbid' in data['metadata']['tags']:
```

Comment:
Yes I think it makes much more sense. We determine the ID type in the endpoint function that can pass it as an argument!